File: TEO/Mav.UI/Mav.Ui/src/app/models/SLT.listing.model.ts
import { UserAssessmentListingModel } from './search/user.assessment.listing.model';

export class SLTListingModel {

  public static default = new SLTListingModel();

  public id: string;

  public firstName: string;

  public surname: string;

  public email: string;

  public enabled: boolean;

  public latestAssessment?: UserAssessmentListingModel;

}

File: TEO/Mav.Models/SearchModels/SLTListingModel.cs
namespace Mav.Models.SearchModels
{
    public class SLTListingModel
    {
        public string Id { get; set; }

        public string FirstName { get; set; }

        public string Surname { get; set; }

        public string Email { get; set; }
        
        public bool Enabled { get; set; }

        public UserAssessmentListingModel LatestAssessment { get; set; }
    }
}
